msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,95 +65,673 @@ You must configure the `fetchTask` property under `bots.configuration` object in
   ],
 ```
 
-For more information, see [public developer preview app manifest schema](../../resources/schema/manifest-schema-dev-preview.md#botsconfiguration).
+For more information, see [app manifest schema](../../resources/schema/manifest-schema.md#botsconfiguration).
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
+When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` or `config/submit`. The bot responds with an Adaptive Card, when the user provides relevant information in the Adaptive Card and selects Submit, `config/submit` or `config/continue` is returned to the bot and the bot configuration is complete.
```

Comment:
Did you confirm that the response **has** to be an adaptive card, and can;t be an iframe-based task module? I don't think that's true, because the `bot.configuration` supposedly takes `taskInfo.url`.